pkgs/tools/security/sget/default.nix

```
@@ -0,0 +1,33 @@
+{ stdenv, lib, buildGoModule, fetchFromGitHub, installShellFiles }:
+
+buildGoModule rec {
+  pname = "sget";
+  version = "unstable-08102022";
```

```suggestion
  version = "unstable-2022-10-08";
```

Also I generally set the date to the date of the commit im pulling, not the date of the change I'm making

---

Yep it's better, thanks for the review!